Jenkinsfile

```diff
@@ -1,13 +1,16 @@
 // Set up the libraries
-@Library('socrata-pipeline-library')
+@Library('socrata-pipeline-library@sarahs/EN-62410/rework-parameters-for-rms-integration') // branch for testing
+
+import com.socrata.ReleaseMetadataService
+def rmsSupportedEnvironment = com.socrata.ReleaseMetadataService.SupportedEnvironment
 
 // set up service and project variables
-def service_server = 'soql-server-pg'
-def project_wd_server = 'soql-server-pg'
-def service_secondary = 'secondary-watcher-pg'
-def project_wd_secondary = 'store-pg'
-def isPr = env.CHANGE_ID != null
-def lastStage
+String service_server = 'soql-server-pg'
```

Comment:
The naming is preexisting, but I think `server_service_name` better aligns with our conventions elsewhere. I won't worry about updating given that such naming conventions aren't relevant to the PR's purpose.